libr/anal/fcn.c

```diff
@@ -532,26 +536,11 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			break;
 		// TODO: use fcn->stack to know our stackframe
 		case R_ANAL_STACK_SET:
-			if ((int)op.ptr > 0) {
```

Comment:
or better fix what is broken in fact If the broken code is more than what I can handle I can just do the merge of af and afCa but still this code it already breaks too much to stay